packages/klighd-core/src/filtering/semantic-filtering-util.ts

```diff
@@ -366,6 +373,74 @@ export namespace NumericNotEqualConnective {
     }
 }
 
+/**
+ * A Numeric Plus Connective takes two numeric operands and evaluates
+ * to their sum.
+ */
+export class NumericPlusConnective implements BinaryConnective {
+    static NAME = "NUMERICPLUS"
+    name = NumericPlusConnective.NAME
+    leftOperand: SemanticFilterRule
+    rightOperand: SemanticFilterRule
+}
+
+export namespace NumericPlusConnective {
+    export function evaluate(conn: NumericPlusConnective): number {
+        return evaluateNumeric(conn.leftOperand) + evaluateNumeric(conn.rightOperand);
+    }
+}
+
+/**
+ * A Numeric Minus Connective takes two numeric operands and evaluates
+ * to their difference.
+ */
+export class NumericMinusConnective implements BinaryConnective {
+    static NAME = "NUMERICMINUS"
+    name = NumericMinusConnective.NAME
+    leftOperand: SemanticFilterRule
+    rightOperand: SemanticFilterRule
+}
+
+export namespace NumericMinusConnective {
+    export function evaluate(conn: NumericMinusConnective): number {
+        return evaluateNumeric(conn.leftOperand) - evaluateNumeric(conn.rightOperand);
+    }
+}
+
+/**
+ * A Numeric Times Connective takes two numeric operands and evaluates
+ * to their product.
+ */
+ export class NumericTimesConnective implements BinaryConnective {
+    static NAME = "NUMERICTIMES"
+    name = NumericTimesConnective.NAME
+    leftOperand: SemanticFilterRule
+    rightOperand: SemanticFilterRule
+}
+
+export namespace NumericTimesConnective {
+    export function evaluate(conn: NumericTimesConnective): number {
+        return evaluateNumeric(conn.leftOperand) * evaluateNumeric(conn.rightOperand);
+    }
+}
+
+/**
+ * A Numeric Divides Connective takes two numeric operands and evaluates
+ * to their product.
+ */
+ export class NumericDividesConnective implements BinaryConnective {
+    static NAME = "NUMERICDIVIDES"
+    name = NumericDividesConnective.NAME
+    leftOperand: SemanticFilterRule
+    rightOperand: SemanticFilterRule
+}
+
+export namespace NumericDividesConnective {
+    export function evaluate(conn: NumericDividesConnective): number {
+        return evaluateNumeric(conn.leftOperand) / evaluateNumeric(conn.rightOperand);
+    }
+}
```

Comment:
I suggest to rename these to …Addition…, …Subtraction…, …Multiplication…, and …Division…